jenkins-jobs/pipelines/build-debezium-tool-images.groovy

```diff
@@ -0,0 +1,37 @@
+import groovy.json.*
+import java.util.*
+
+IMAGES_DIR = 'images'
+GIT_CREDENTIALS_ID = 'debezium-github'
+DOCKER_CREDENTIALS_ID = 'debezium-dockerhub'
+
+node('Slave') {
+    try {
+        stage('Initialize') {
+            dir('.') {
+                deleteDir()
+            }
+            checkout([$class                           : 'GitSCM',
+                      branches                         : [[name: IMAGES_BRANCH]],
+                      doGenerateSubmoduleConfigurations: false,
+                      extensions                       : [[$class: 'RelativeTargetDirectory', relativeTargetDir: IMAGES_DIR]],
+                      submoduleCfg                     : [],
+                      userRemoteConfigs                : [[url: "https://$IMAGES_REPOSITORY", credentialsId: GIT_CREDENTIALS_ID]]
+            ]
+            )
+            withCredentials([usernamePassword(credentialsId: DOCKER_CREDENTIALS_ID, passwordVariable: 'DOCKER_PASSWORD', usernameVariable: 'DOCKER_USERNAME')]) {
+                sh """
+                    docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+                """
+            }
+        }
+        stage('master') {
+            echo "Building debezium tool images"
+            dir(IMAGES_DIR) {
+                sh "PUSH_IMAGES=true TAG=$TAG ./build-tool-images.sh"
+            }
+        }
+    } finally {
+        mail to: 'jpechane@redhat.com', subject: "${JOB_NAME} run #${BUILD_NUMBER} finished", body: "Run ${BUILD_URL} finished with result: ${currentBuild.currentResult}"
```

Comment:
Recipient email address should be a var like `IMAGES_DIR` at the top?

Reply:
@ani-sha That's good proposal. Could you please create a separate Jira and do it as a followup for all jobs?
Also could you please get in touch with @jcechace as it seems theat `currentBuild.currentResult` is `SUCCESS` even in case of failure?

Reply:
Logged https://issues.redhat.com/browse/DBZ-4071 for the same.